.github/workflows/rust.yml

```diff
@@ -80,3 +80,25 @@ jobs:
           DO_FEATURE_MATRIX: true
         run: ./contrib/test.sh
 
+  Embedded:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        run: sudo apt install qemu-system-arm
```

Comment:
You usually want to run an `apt update` or `apt-get update` immediately before an install to make sure you're pulling the latest versions (which is why you're getting 404s in CI now).

Reply:
right, thanks